site/releases/2024-dec-19/release-notes.qmd

```diff
@@ -0,0 +1,411 @@
+Labels: bug
+--->
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+- Previously, deleting a finding did not remove it from linked validation reports. 
+- Now, deleting a finding will show a confirmation warning and subsequently remove it from any linked reports.
+
+::: {.tc}
+[Add and manage model findings](/guide/model-validation/add-manage-model-findings.qmd#delete-model-findings){.button}
+
+:::
+
+:::
+
+::: {.w-40-ns .tc}
+![Deletion confirmation dialogue for a finding linked to a validation report](delete-linked-finding.png){fig-alt="A screenshot of the deletion confirmation dialogue for a finding linked to a validation report" .screenshot}
+
+:::
+
+::::
+
+## Documentation (v1.27.17)
+
+### Code samples
+
+#### Initialize {{< var vm.product >}} with stored credentials
+
+We've included details on how to initialize {{< var vm.product >}} with credentials stored in an `.env` file in our Jupyter Notebook samples.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+Our documentation guides have also been updated to match this new experience:
+
+:::
+
+::: {.w-40-ns .tc}
+[Store model credentials in `.env` files](/developer/model-documentation/store-credentials-in-env-file.qmd){.button}
+
+:::
+
+::::
+
+<!---
+PR #543: Add developer fundamentals videos & training fixes
+URL: https://github.com/validmind/documentation/pull/543
+Labels: documentation
+--->
+### User guide updates
+
+#### Developer Fundamentals video series
+
+We created a series of 10 short videos to help you learn about the model documentation process as a developer:
+
+- These videos introduce you to generating model documentation that includes tests. 
+- You will also learn about adding your own tests, editing the content online, and then submitting your documentation for approval so that a validator can review your submitted documentation.
+
+{{< video https://www.youtube.com/embed/videoseries?si=aP7yLOGO60h4vi8C&amp;list=PL7I7ZrCoVCCP876aLRCjitZqYif96Whlz title='Developer Fundamentals' >}}
+
+<!---
+PR #552: Updated workflows/actions to dynamically generate an `.env` file
+URL: https://github.com/validmind/documentation/pull/552
+Labels: documentation
+--->
+
+<!-- https://github.com/validmind/documentation/pull/567 -->
+
+### {{< var validmind.training >}} updates
+
+##### Example executed Jupyter Notebook
+
+::: {.column-margin}
+::: {.preview source="/training/developer-fundamentals/developer-fundamentals.qmd"}
+:::
+
+:::
+
+The {{< var validmind.training >}} **Developer Fundamentals** course has been updated with an improved version of the **{{< var vm.product >}} Introduction for Model Developers** Jupyter Notebook: 
+
+- This embedded notebook is now executed live within the training. This allows you to interact with output cells previously omitted, such as when previewing the documentation template.
+- This training notebook serves as a reference guide. You will understand what to expect when you first run the cells.
+
+### Site enhancements
```

Comment:
I'm not gonna stop you from tooting our own horn here but I wouldn't be entirely surprised if we get asked to reduce the amount of detail here. (A LOT of work has gone into our docs site recently, especially from your end, so it's nice to see this work reflected in the release notes.)

Reply:
In my defense, we NEVER highlighted these and it's only with the last two releases we've actually put any attention into them. I don't see why we're not allowed to talk about the neat things we work on site wise? Docs are part of done, after all.

Reply:
> I don't see why we're not allowed to talk about the neat things we work on site wise?

We’ve included updates about the docs site in the past because they are worthwhile. I fully support that.

What I’m suggesting is that the level of detail here might be too much. Release notes should focus on what’s useful for users to _do something with the product_. While the 'after' view of the docs site changes helps orient readers, the 'before' details are unnecessary. Users won’t encounter that version again—it’s interesting historically but not relevant for getting started with this release.

Succinctly: release notes should be the tip of the iceberg. Historical details that don’t impact what users can do now aren’t very useful.

(If you're OK with it, let's maybe move the rest of this conversation to our community standup later this afternoon?)